### ISSUE TITLE
Save only DRB data in a target for large CONUS downloads

### ISSUE THREAD

**Issue by jds485** [USGS-R/drb-inland-salinity-ml#77]
This issue is to reduce the storage size of downloaded data in our repo. The plan based on our meeting is to combine 3 steps into one target that calls the existing functions for: 
1. the fetch of the zipped file, 
2. unzipping, and 
3. clipping to DRB 
4. [new step] deleting the zipped and unzipped files. 

The new target would save the clipped dataset.

Examples: NADP and NLCD land cover datasets

**Comment by jds485** [comment]
Lauren suggests applying unzip and clip function to NADP and NLCD datasets

**Comment by jds485** [comment]
After #148, I think only NLCD datasets would need to be clipped

**Comment by jds485** [comment]
Also applies to road salt rasters

**Comment by jds485** [comment]
Total storage of these files is < 1 GB, so this is low priority
